src/index.js

```diff
@@ -2,12 +2,32 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Main from '../src/components/Main';
+import TVDetail from '../src/components/TVDetail';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Routes>
+        <Route path='/' element={ <App /> } >
+          <Route index element={ <Main/> } />
+          <Route path='Shows' element={ <Main/> } >
+            <Route path=':number' element={<TVDetail />} />
+          </Route>
+          <Route path='*' element={
+            <div style={{
+              padding: '1rem'
+            }}>
+              <p>Still Building...</p>
+            </div>
+          }
+          />
+        </Route>
+      </Routes>
+    </BrowserRouter>
```

Comment:
This is an incorrect code place, you should put your router setup logic into an independent file under the root directory.  You can call the file name router.js.

**index.js only for mounting react application**